data_structure/tests/test_array.py

```diff
@@ -0,0 +1,75 @@
+import unittest
+from data_structure.array import *
+
+
+class TestPlusOne(unittest.TestCase):
+    def test_plusOne_one(self):
+        output = PlusOne().plusOne([1, 2, 9])
+        ans = [1, 3, 0]
+        self.assertEqual(ans, output)
+
+    def test_plusOne_all_carry_over(self):
+        output = PlusOne().plusOne([9, 9, 9])
+        ans = [1, 0, 0, 0]
+        self.assertEqual(ans, output)
+
+    def test_plusOne_two(self):
+        output = PlusOne().plusOne([0])
+        ans = [1]
+        self.assertEqual(ans, output)
+
+
+class TestAddBinary(unittest.TestCase):
+    def test_addBinary_one(self):
+        output = AddBinary().addBinary('11', '1')
+        ans = "100"
+        self.assertEqual(ans, output)
+
+    def test_addBinary_two(self):
+        output = AddBinary().addBinary('1010', '1011')
+        ans = "10101"
+        self.assertEqual(ans, output)
+
+
+class TestShortestToChar(unittest.TestCase):
+    def test_ShortestToChar(self):
+        output = ShortestToChar().shortestToChar("loveleetcode", 'e')
```

Comment:
it's not clear what this function do to a reviewer. Typically, your function should explain what it does, but if what it does is too complex to explain in the function name, then you should do inline comment briefly explaining what you expect it to do. Same goes for all the unit test cases below this one (maybe the ones above too, probably).

if you are not entirely sure how to do it, the `given, when, then` structure is a good start. Read https://martinfowler.com/bliki/GivenWhenThen.html for inspirations